spec/features/phone/add_phone_spec.rb

```diff
@@ -92,16 +92,16 @@
     sign_in_and_2fa_user(user)
     click_on "+ #{t('account.index.phone_add')}"
 
-    expect(page.find('label', text: 'Text message (SMS)')[:class]).to_not include('btn-disabled')
-    expect(page.find('label', text: 'Phone call')[:class]).to_not include('btn-disabled')
+    expect(page.find('label', text: 'Text message (SMS)')).to_not match_css('.usa-button--disabled')
+    expect(page.find('label', text: 'Phone call')).to_not match_css('.usa-button--disabled')
```

Comment:
these are great matcher cleanups!

Reply:
> these are great matcher cleanups!

Motivated in part because the previous implementation would have exceeded Rubocop's acceptable line length 😄